packages/scss/src/components/_form.scss

```diff
@@ -31,73 +40,145 @@
 
 .form {
 	&.mod-framed {
-		.form-group {
+		.form-group-line {
 			display: flex;
-			flex-direction: column;
+			position: relative;
 		}
 
-		.form-group-line {
+		.form-group-line-col {
+			flex-basis: 0;
+			flex-grow: 1;
 			display: flex;
-			flex: 0 1 auto;
-			flex-direction: row;
-			flex-wrap: wrap;
-			position: relative;
+			flex-direction: column;
 
-			&.mod-stacked {
-				flex-direction: column;
+			> * {
+				flex-grow: 1;
 			}
 
-			&.mod-selection {
-				@include fakeBorder(1px, _theme("commons.divider.color"));
-				padding: _component("field.framed.top-padding") _component("field.framed.side-padding") _component("field.framed.bottom-padding");
+			&.mod-half {
```

Comment:
Why a half/third/... rather than the 12col system ? It offers more flexibility

Reply:
I thought that a grid offered too many possibilities here and that in practice only very few cases were used. I thought that a grid has more its place at the document level, and not in a nested component.

Reply:
It might offer more than needed, but it is more consistent on how we handle cols on a line and it offers breakpoint handling (for example being 50/50 on a widescreen and 100% on handheld device)

Reply:
Okay, I'll put this back on.

Reply:
👌